modules/ROOT/pages/configure-openshift-routes.adoc

```diff
@@ -0,0 +1,99 @@
+= Configuring Red Hat OpenShift Routes in Runtime Fabric
+
+Runtime Fabric enables you to configure Red Hat OpenShift Routes to create ingress endpoints for your Mule applications. 
+
+== Before You Begin
+
+Configuration of routes in Runtime Fabric follows closely to ingress resource configuration. Before you create routes, review the following:
+* xref:custom-ingress-configuration.adoc#how-ingress-resources-templates-work-in-runtime-fabric[How Ingress Resources Templates Work in Runtime Fabric] 
```

Comment:
I think you need an empty line here to make the bullet list render properly. In the GH preview it doesn't show as a bullet list, please check in local build.